client/scripts/Player.js

```diff
@@ -0,0 +1,284 @@
+class Player {
+	constructor() {
+		this.isHost = false
+		this.isFakeArtist = false
+		this.socket = io()
+		this.listOfPlayers = null
+		this.room_id = null
+		this.color = null
+		this.number = null
```

Comment:
since your constructor doesnt take any args, these initializations can happen out of the constructor:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Public_class_fields#public_instance_fields